brands/amenity/cinema.json

```
@@ -256,6 +256,15 @@
       "name": "Cinéville"
     }
   },
+  "amenity/cinema|Megarama": {
+    "locationSet": {"include": ["fr"]},
+    "tags": {
+      "amenity": "cinema",
+      "brand": "Megarama",
+      "brand:wikidata": "Q96707027",
+      "name": "Megarama Cinemas"
```

Just "Megarama" is ok I think.
```suggestion
      "name": "Megarama"
```